site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,112 @@
+----
+[#setting url_escaping_charset="UTF-8"]
+To change your password click on the following link.
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+
+${url}
+
+- FusionAuth Admin
+----
+
+[source,text,title="The text email template content"]
+----
+[#setting url_escaping_charset="UTF-8"]
+To change your password click on the following link.
+
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+
+${url}
+
+- FusionAuth Admin
+----
+
+Save your template by clicking the blue save icon.
+
+When a user clicks the `Forgot Password` link on the login page or calls the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[forgot password API], this template will be used to build the email sent to the user. This is the start of the workflow to change their password. This template is used so long as the application doesn't have a `Forgot Password` template configured.
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview]. After, log in to your application by clicking on `Applications` in the navigation sidebar, selecting the green magnifying glass icon, and copying and pasting the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+=== Configuring Application Email Templates
+
+Finally, you'll override your generic Tenant Email Template you just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the blue edit icon for the `Forgot Password` email template and copy the `HTML Template` `Text Template` sections into a notepad.
+
+Create a new email template by going to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
```

Comment:
Okay, I am going to say `Name`, `Default Subject`, `From Email`, `Default from Name`, `Default HTML`, and `Default Text` using the [field][#oijdf#format#. Going to leave `HTML Template` and `Text Template` in backticks because they are technically links, not fields. @mooreds